build.gradle.kts

```diff
@@ -64,3 +64,7 @@ spotless {
         ktlint()
     }
 }
+
+tasks.named("build") {
+    dependsOn(":spotlessApply")
```

Comment:
❤️

Reply:
this peace of love is already merged into master